README.md

```diff
@@ -11,10 +11,13 @@ Please follow the instructions of that repo which is also written below.
 
 ===================================================
 
+In permutohedral_lattice/CMakeLists.txt, set the path to the **libtensorflow_framework.so** file in line 8.
+
 To compile the code run:
 ````
 sh build.sh
 ````
+Running the above command in the main folder gives you **lattice_filter.so** file.
```

Comment:
Why are we mentioning this? Does the user have to know the path of this file? (Or existence of this file)

Reply:
Yup. I wanted to make sure that the file exists, otherwise our code would give an error.